hector_exploration_node/src/exploration_node.cpp

```diff
@@ -39,7 +39,7 @@ class SimpleExplorationPlanner
   SimpleExplorationPlanner()
   {
     ros::NodeHandle nh;
-
+    buffer_.setUsingDedicatedThread(true)
```

Comment:
This is not going to compile;

Reply:
Thanks for the catch, the semicolon typo has been fixed.